packages/nextjs/app/page.tsx

```diff
@@ -1,25 +1,94 @@
+    description:
+      "I'm passionate about blockchain and novel technologies. I'm studying ZKPs. Hit me up if you want to chat about it!",
+    github: "mufasa-0420",
+    telegram: "bobbyliu0420",
+  },
+  {
+    address: "0x5c3f1e2a6b7d8f9e3c4a5b6c7d8e9f0a1b2c3d4e",
+    name: "Cool Guy",
+    avatar: "https://avatars.githubusercontent.com/u/12345678?v=4",
+    description:
+      "👋 Hi! I’m CoolGuy, a passionate builder focused on smart contracts, DeFi, and open source. I love learning, collaborating, and shipping unstoppable apps!",
+    github: "coolguy0525",
+    telegram: "coologuy0525",
+  },
+  {
+    address: "0x645b4605dC74e8eFF38d3FD26C3cabA853ABbF61",
+    name: "Daniel",
+    avatar: "https://avatars.githubusercontent.com/u/87654321?v=4",
+    description: `Hi, I'm Daniel, a full-stack developer with a foundation in JavaScript based stacks. I've been
+          interested in web3 development for a while now and decided to start out my journey with BuidlGuidl's
+          SpeedRunEthereum. I enjoy the challenge of building complete applications from start to finish and I look
+          forward to leveraging my skills by contributing to interesting and innovative decentralized applications.`,
+    github: "otomdee",
+    telegram: "Ox_Money",
+  },
+  {
+    address: "0xba230e7c7E39D09443d8Da0a332DD787BD1352cb",
+    name: "ankur-JA",
+    avatar: "https://avatars.githubusercontent.com/u/11223344?v=4",
+    description: `I am passionate about building modern web applications and decentralized applications (dApps). I enjoy
+          exploring new blockchain technologies, contributing to open-source projects, and learning about cybersecurity
+          and smart contract development.`,
+    github: "ankur-JA",
+    linkedin: "ankur-raj-gearhead",
+  },
+  {
+    address: "0xdaC6E56F86Fb6f18c4114C2E4f41901ab9803096",
+    name: "Nejc",
+    avatar: "https://avatars.githubusercontent.com/u/22334455?v=4",
+    description: `I am a blockchain enthusiast and a researcher exploring the connections between mechatronics and
+              blockchain technology. Hit me up if you want to chat about it!`,
+    github: "NejcRozman",
+    telegram: "NRosca",
+  },
+  {
+    address: "0xE474023791245Ff654167397Ea8262B891Ba207c",
+    name: "Daniel",
+    avatar: "https://avatars.githubusercontent.com/u/87654321?v=4",
+    description: `Hi, I'm Daniel, a full-stack developer with a foundation in JavaScript based stacks. I've been
+          interested in web3 development for a while now and decided to start out my journey with BuidlGuidl's
+          SpeedRunEthereum. I enjoy the challenge of building complete applications from start to finish and I look
+          forward to leveraging my skills by contributing to interesting and innovative decentralized applications.`,
+    github: "otomdee",
+    telegram: "Ox_Money",
+  },
+  // ...add avatars for other members as needed...
+];
```

Comment:
Ox\_Money is mentioned twice, and you forgot neerajnerlekar and adashima152. I think it would be better to fetch this information programmatically to make sure no one gets left out.